modules/programs/rbw.nix

```diff
@@ -52,15 +51,15 @@ let
             ```nix
             services.dbus.packages = [ pkgs.gcr ];
             ```
-            For this reason, the default is `gtk2` for
-            now.
           '';
           # we want the program in the config
           apply = val:
             if builtins.isString val then
               "${pkgs.pinentry.${val}}/bin/pinentry"
+            else if val ? binaryPath then
+              "${val}/${val.binaryPath}"
             else
-              "${val}/${val.binaryPath or "bin/pinentry"}";
+              lib.getExe val;
```

Comment:
I'm not sure if we could migrate `pinentry_mac` to define `meta.mainProgram`, that could remove the `binaryPath` part and rely on `lib.getExe` for every package.

Reply:
See [this PR]() which adds it to `pinentry_mac` (and others).

See [that PR]() which adds it to the various `pinentry.<flavor>` packages.

So we could further improve this once both PRs are merged, though it shouldn't block this one.